README.md

```diff
@@ -9,188 +9,44 @@ maintainer: groenborg
-```
-
-_Then install using scoop_:
-```
-scoop install git-phlow
-```
-
-**Install using Chocolatey**
-
-Package is available here, also including installation command: https://chocolatey.org/packages/git-phlow/
-
-You need to have Chocolatey installed, and then just run the following command in powershell:
-
-    choco install git-phlow
+- *works with jira*: sets assignee and does transition
 
+- *branches based on issues*: create a workspace from an issue to ensure traceability
 
-##### Compatibility
-- Powershell v5.1+
-- Command prompt - only with the `--no-color` flag
+- *fully automatable*: `deliver` your branches, ready for your pipeline to integrate, test and merge.
 
-## Updating git-phlow (tool)
-### Mac
-On Mac using homebrew, run the upgrade command.
-
-```shell
-brew upgrade git-phlow
-```
-### Linux
-In linux you run the installation again, but this time override the old `git-phlow` binary.
-
-### Windows
-Using either Scoop or Chocolatey you installed from originally run:
-```
-scoop update git-phlow
-```
-or
-```
-choco upgrade git-phlow
-```
-_Please notice Chocolatey packages can wait on approval for being published, so there might be waiting time for latest version._
-
-## Compatability Matirix
-
-|                | support | version | 
-| -------------- | ------- | ------- |
-| GitHub         | &#x2714;| v3
-| GitHub Premise | &#x2715;| -
-| Jira Cloud     | &#x2715;| -
-| Jira Premise   | &#x2714;| 7.3.0
-|**macOS**       |
-| bash           | &#x2714;| 3.2.57
-| zsh            | &#x2714;| 5.4.2
-| **windows**    |
-| [git bash](https://git-for-windows.github.io)| &#x2714;| 4.4.12 |
-| powershell     | &#x2714;| 5.1.14393  |
-| cmd            | &#x2714;| 10.0.14393 |
-| [cmder](http://cmder.net)| &#x2715;  | - |
-
-
-## Usage
-- First time, authenticate to GitHub:
-
-```git
-git phlow auth
-```
-
-- Look at the issues by using `ghi` or going to GitHub or Waffle
-- Find an issue to work on, and note its id e.g. `42`
-- Go to the local git repository and type:
-
-```git
-git phlow workon 42
-```
-
-- This will create a branch: `42-some-issue-title`
-- Make the changes and complete the task
-- When done, add everything and commit with a single command:
-
-```git
-git phlow wrapup
-```
-
-- Now that the changes are ready, type:
-
-```git
-git phlow deliver
-```
-
-- It renames the local branch to: `delivered/42-some-issue-title`
-- It pushes remote branch named: `ready/42-some-issue-title`
+## Getting started
 
-If the automation tools have been configured to listen for __ready/__ branches, they will pick up the branch and integrate them with the stable branch.
-If you do not have an automation system, you can deliver the work to your default branch by doing a local deliver instead.
-
-```git
-git phlow deliver --local
-```
+To get started using git-phlow, read [praqmas](https://www.praqma.com/stories/git-phlow/) article about how git phlow works and consult the [official documentation](/docs/README.md)
```

Comment:
This could be changed to: 
To read the concepts behind git-phlow read [praqma's article](https:....) about the workflow.
Consult the [official documentation](/docs/README.md) for information on how to get started using git-phlow